### PR TITLE
Update TESTING.md instructions

### PR DESCRIPTION
**Pull request by jasonblais**
Before submitting, please confirm you've
 - [X] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
 - [X] completed [Mattermost Contributor Agreement](http://www.mattermost.org/mattermost-contributor-agreement/)
 - [N/A] executed `npm run lint:js` for proper code formatting

Please provide the following information:

**Summary**

Update TESTING.md instructions

### CODE REVIEW

File: TESTING.md [hunk 1]
```diff
@@ -38,10 +53,3 @@ The following chart summarizes the operating systems which the Mattermost Deskto
 ## Testing Guides by Operating System
```
Comment (yuya-oc) [inline TESTING.md]: Now there is only a section header. Should this be left?

Comment (jasonblais) [inline TESTING.md]: Thanks @yuya-oc, you're right. No need to have this

**Comment by yuya-oc** [comment]
I feel it's ready to merge.

@jasonblais To make sure, would you confirm again? If it's okay, please merge.

**Comment by jasonblais** [comment]
Thanks @yuya-oc!

**Pull request merged**